wagtail/wagtailadmin/tests/test_pages_views.py

```diff
@@ -4068,3 +4068,62 @@ def test_field_and_non_field_error(self):
         self.assertContains(response, """<p class="error-message"><span>This field is required.</span></p>""", count=1, html=True)
         # Error on title shown in the header message
         self.assertContains(response, "<li>Title: This field is required.</li>", count=1)
+
+
+class TestDraftAccess(TestCase, WagtailTestUtils):
+    """Tests for the draft view access restrictions."""
+
+    def setUp(self):
+        # Find root page
+        self.root_page = Page.objects.get(id=2)
+
+        # Add child page
+        self.child_page = SimplePage(
+            title="Hello world!",
+            slug="hello-world",
+            content="hello",
+        )
+        self.root_page.add_child(instance=self.child_page)
+
+        # create unpriviliged user
```

Comment:
I'm going to phrase this differently, as the user is priviliged in the sense of having admin access, but not priviliged enough to see the draft.